tests/bootstrap.php

```diff
@@ -5,4 +5,26 @@
  */
 
 $loader = require __DIR__ . "/../vendor/autoload.php";
-$loader->addPsr4( 'YOURLS\\', __DIR__ );
+
+
+/**
+* Returns the name of a temporary file
+*/
+function temp_filename() {
+    $tmp_dir = realpath(__DIR__);
+    return tempnam($tmp_dir, 'pomo');
+}
+
+function replace_r_n($text) {
```

Comment:
Otherwise tests locally fail on Windows on these assertions: https://github.com/LeoColomb/pomo/blob/61accd8492309a105b8494401f8a61af4777690d/tests/POTest.php#L113-L119